lib/routing.js

```diff
@@ -43,6 +43,11 @@ exports.routes = [
     config: require('./routes/client/get')
   },
   {
+    method: 'POST',
+    path: v('/client/key-data/{client_id}'),
+    config: require('./routes/key_data')
```

Comment:
You moved the `.js` route implementation out from under the `clients` directory, but it's still namespaced under "/client/" in the API, which seems like a mis-match.  Should it be e.g. just "/key-data" and accept the the "client_id" as an additional parameter in the `POST` body?

Reply:
> Should it be e.g. just "/key-data" and accept the the "client_id" as an additional parameter in the POST body?

Yeah I guess after moving it a bit that makes sense 👍